doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -741,6 +742,21 @@ As a fallback mechanism for clock synchronization, the accessory must simultaneo
 The Fast Pair advertising frames make the affected Provider visible to nearby Seekers.
 Once one of the Seeker devices connects to the accessory and synchronizes the clock, the :c:member:`bt_fast_pair_fmdn_info_cb.clock_synced` callback is called to indicate that the Provider is no longer required to advertise the Fast Pair payload.
 
+The connected peer authentication is indicated by the :c:member:`bt_fast_pair_fmdn_info_cb.conn_authenticated` callback.
+This callback is triggered on a successful provisioning state read operation over the Beacon Actions characteristic.
+A successful operation proves that the connected peer has at least one of the Provider's Account Key.
+
+A typical use case for this callback is to authenticate the connected peer and allow them to read the firmware version to support firmware update intents on the Android platform.
+The firmware version can be read over the Firmware Revision characteristic in the Device Information Service (DIS).
+It is assumed that this characteristic contains the identifying information and should not be readable in all circumstances to prevent tracking.
+The alternative authentication mechanism can be used if the defined device model from the Google Nearby Devices console supports Bluetooth bonding during the Fast Pair procedure.
```

Comment:
```suggestion
The alternative authentication mechanism can be used if the defined device model from the Google Nearby Devices console does not support Bluetooth bonding during the Fast Pair procedure.
```

Reply:
I described the authentication flow with the callback.

This sentence is about notifying the user about the alternative authentication flow for devices that DO bond.

Reply:
btw. Shouldn't it be `An alternative authentication mechanism ...` if we are introducing a new mechanism here?